static/js/components/SiteContentEditor.tsx

```diff
@@ -73,26 +103,35 @@ export default function SiteEditContent(props: Props): JSX.Element | null {
     }
     setSubmitting(false)
 
+    // refresh to have the new item show up in the listing
+    repeatListingRequest()
```

Comment:
Is it necessary to hit this endpoint again if we're just editing an item? I was under the impression that we update the relevant redux query entities after an edit, so the listing page should automatically have up-to-date information for the edited item since it uses the same entities. If that's true, the API call wouldn't be necessary

Reply:
yeah I think it's not strictly necessary, because if you check the sequence of updates to the store the new version of the site data is propagated to the store when the edit request comes back successfully. So we could just lean on that to ensure that the view on the content list remains consistent.

thinking about it more, though, just dropping a refresh on the listing API does protect us from some degree of inconsistency that could arise if multiple people are working on content at the same time (not really on the piece of content just edited, but on others). So I'm somewhat inclined to leave it around as 'insurance', if you will...idk, what do you think? It's a (very) small increase in load on the server, but my understanding is that ~5-10 users are going to be ever using this at the same time, so we're really not going to be dealing with scaling headaches here.

Reply:
Sure, we can leave it